addons/sale_timesheet/models/product.py

```diff
@@ -24,15 +24,22 @@ def _selection_service_policy(self):
     service_upsell_threshold = fields.Float('Threshold', default=1, help="Percentage of time delivered compared to the prepaid amount that must be reached for the upselling opportunity activity to be triggered.")
     service_upsell_threshold_ratio = fields.Char(compute='_compute_service_upsell_threshold_ratio')
 
-    @api.depends('uom_id')
+    @api.depends('uom_id', 'company_id')
```

Comment:
Why does it have to depend on company?

Reply:
Because Product might have different company and company have different `timesheet_encode_uom_id` :slightly_smiling_face:

Reply:
Oops, I didn't see that